frontend/src/app/types.d.ts

```diff
@@ -0,0 +1,12 @@
+type UserType = "standard" | "restricted" | "admin";
```

Comment:
Can we move all these to `permissions.tsx`? Global anything is a pain to find. If we have to import these in a lot of places that's more a problem with the way we're using them than with the lack of global variables. In many cases TS will figure out literal strings like these without imports anyway.